integration-tests/src/test/java/integration/MongoDbSeed.java

```diff
@@ -25,5 +25,5 @@
 @Retention(RetentionPolicy.RUNTIME)
 public @interface MongoDbSeed {
     String[] locations() default {};
-    String database() default "graylog2";
+    String database() default "";
```

Comment:
The default should probably be `IntegrationTestsConfig.mongodbDatabase`, shouldn't it?